docs/tutorialkit.dev/src/content/docs/reference/theming.mdx

```diff
@@ -0,0 +1,406 @@
+
+| Token                                                  | Description                                             |
+| ------------------------------------------------------ | ------------------------------------------------------- |
+| `--tk-elements-topBar-iconButton-backgroundColor`      | The background color of the icon button.                |
+| `--tk-elements-topBar-iconButton-backgroundColorHover` | The background color of the icon button when hovering.  |
+| `--tk-elements-topBar-iconButton-iconColor`            | The color of the icon.                                  |
+| `--tk-elements-topBar-iconButton-iconColorHover`       | The color of the icon when hovering.                    |
+
+
+### Panels
+
+Panels are containers which can be resized and can hold different kind of components. For instanceof, the file tree, or the terminal.
+
+| Token                                      | Description                         |
+| ------------------------------------------ | ----------------------------------- |
+| `--tk-elements-panel-textColor`            | The text color of the panels.       |
+| `--tk-elements-panel-backgroundColor`      | The background color of the panels. |
+
+#### Header
+
+The panel header is the top part of the panel that contains the title, an icon, and optionally an action button
+
+![Panel Header](./images/theming-panel-header.png)
+
+| Token                                        | Description                          |
+| -------------------------------------------- | ------------------------------------ |
+| `--tk-elements-panel-header-backgroundColor` | The background color of the header.  |
+| `--tk-elements-panel-header-textColor`       | The text color of the header.        |
+| `--tk-elements-panel-header-iconColor`       | The color of the icon in the header. |
+
+##### Button
+
+The button in the header is shown at the right side of the header of the panel.
+
+![Panel Header Button](./images/theming-panel-header-button.png)
+
+| Token                                                   | Description                                       |
+| ------------------------------------------------------- | ------------------------------------------------- |
+| `--tk-elements-panel-headerButton-textColor`            | The text color of the button.                     |
+| `--tk-elements-panel-headerButton-textColorHover`       | The text color of the button when hovering.       |
+| `--tk-elements-panel-headerButton-backgroundColor`      | The background color of the button.               |
+| `--tk-elements-panel-headerButton-backgroundColorHover` | The background color of the button when hovering. |
+| `--tk-elements-panel-headerButton-iconColor`            | The icon color.                                   |
+| `--tk-elements-panel-headerButton-iconColorHover`       | The icon color when hovering.                     |
+
+##### Tab
+
```

Comment:
Other sections contain a brief description. Shall we add one here too just for consistency?